src/test/ui/const-generics/unused-substs/unused-substs-4.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(const_generics)] //~ WARN the feature `const_generics` is incomplete
+
+fn bind<const N: usize>(value: [u8; N]) -> [u8; 3 + 4] {
```

Comment:
can be make a test with `bind<const N: usize>([u8; N]) -> [u8; N]`? This should also work.

We should also have a test for `N -> N+1`, which ought not to work.